### PR TITLE
update

### PR DESCRIPTION
**Pull request by nileshhazra**

### CODE REVIEW

File: Warmup/assignment.md [hunk 1]
```diff
@@ -8,3 +8,4 @@ Problem| solved | Date
 --- | --- | --- | 
 https://www.spoj.com/problems/DOL/ |  |  |  |
 
+:
```
Comment (Astrophilic) [inline Warmup/assignment.md]: Remove this colon and put it in appropriate place.

Comment (Astrophilic) [inline Warmup/assignment.md]: Remove this colon and put it in appropriate position.

Comment (nileshhazra) [reply Warmup/assignment.md]: Changes Commited

Comment (Astrophilic) [reply Warmup/assignment.md]: @nileshhazra, Let me look into it.!

**Comment by nileshhazra** [comment]
Changes done

**Comment by nileshhazra** [comment]
clone removed

**Comment by nileshhazra** [comment]
@Astrophilic  I have put the colon in appropriate position

**Comment by Astrophilic** [comment]
ok accepted!

**Pull request merged**